### PR TITLE
Update setup.txt

### PR DESCRIPTION
**Pull request by DandelionSprout**
I have virtually no idea if I'm doing this right, or how long it will take the repository maintainers to see the six pull requests I'll (attempt to) make, with this being the first. I'm just hoping that I can brush up the Norwegian translation of Unchecky somewhat.

### CODE REVIEW

**Comment by RaMMicHaeL** [comment]
Thank you for the translation update!
The program translation looks OK, but the `nb_no.js` file looks scrambled. Could you please fix it?

**Comment by DandelionSprout** [comment]
I'm hoping that this fix should solve it well enough.

**Comment by RaMMicHaeL** [comment]
Yes, that did it. I've merged the pull request, and updated the website with the new translation. Thanks again!

File: program/Norwegian, Bokm%C3%A5l (Norway)/app.txt [hunk 1]
```diff
@@ -4,64 +4,64 @@ IDS_OK                  "OK"
...
 IDS_MAINCTRL_TEXT_QUERYERROR 
-                        "En feil oppstod under etterspørselen for Uncheckys status-tjeneste."
+                        "En feil oppstod under etterspørringen etter Uncheckys statustjeneste."
 IDS_MAINCTRL_STATS_NONE_NEWGUY 
-                        "Ser ut til at Unchecky nettopp er installert. Det er ingen resultater å vise enda."
-IDS_MAINCTRL_STATS_NONE "Ingen sjekkbokser har blitt utkrysset enda. Ser ut til at du holder deg bort fra medfølgende installasjoner med potensielt uønskede programmvare, bra jobba!"
-IDS_MAINCTRL_STATS      "Siden %s, har %u sjekkbokser blitt utkrysset%s."
+                        "Det ser ut til at Unchecky nettopp er blitt installert. Det er ingen resultater å vise ennå."
+IDS_MAINCTRL_STATS_NONE "Ingen sjekkbokser har blitt utkrysset ennå. Ser ut til at du holder deg bort fra medfølgende installasjoner med potensielt uønskede programmer, bra jobbet!"
+IDS_MAINCTRL_STATS      "Siden %s, har %u sjekkbokser blitt utkrysset %s."
```
Comment (RaMMicHaeL) [inline program/Norwegian, Bokm%C3%A5l (Norway)/app.txt]: The added space looks like a mistake. I'll revert it.

**Pull request merged**